packages/block-library/src/footnotes/index.php

```diff
@@ -160,9 +160,8 @@ function wp_add_footnotes_revisions_to_post_meta( $post ) {
 	}
 }
 
-foreach ( array( 'post', 'page' ) as $post_type ) {
-	add_action( "rest_after_insert_{$post_type}", 'wp_add_footnotes_revisions_to_post_meta' );
-}
+add_action( 'rest_after_insert_post', 'wp_add_footnotes_revisions_to_post_meta' );
+add_action( 'rest_after_insert_page', 'wp_add_footnotes_revisions_to_post_meta' );
```

Comment:
Worth noting that we could move this to the `register_block_core_footnotes` function, which runs on `init`, but I'm scared to move code around at this point. It's an option though.

Reply:
We anyway went for hardcoding these for this release.

Reply:
This is fine for now; it'll have to change anyway once we add support for other post types so we can leave bigger changes till then 😄